### PR TITLE
Add molecular use cases

### PR DESCRIPTION
**Pull request by njbruce**
Added first complete for testing molecular use case. Added place holders for others.

### CODE REVIEW

File: src/assets/config_files/usecases.json [hunk 1]
```diff
@@ -1,19 +1,6 @@
 [
   {
     "molecular": [
-      {
-        "title": "Set up a Brownian dynamics simulation for calculating protein-protein association rate constants",
-        "description": "Starting from a structure of the complex formed between adenylyl cyclase 5 and Gαolf obtained from a molecular dynamics simulation, set up all the files required to perform a Brownian dynamics simulation with SDA.",
-        "experience": ["all"],
-        "maturity": ["beta"],
-        "disabled": true,
```
Comment (antonelepfl) [inline src/assets/config_files/usecases.json]: As this is disabled is better to put the available on the top. Just small thing

File: src/assets/config_files/usecases.json [hunk 2]
```diff
@@ -33,10 +20,23 @@
             "extension": ".ipynb",
             "file": "3D6ffd5ebb-a36c-4fa4-9f44-d329c9e6d3e1",
```
Comment (antonelepfl) [inline src/assets/config_files/usecases.json]: I don't have access to that file. Is it in a private Collab?
If that is the case you have 2 options: either add us as team members (@alex4200 , me, @lbologna etc) or create a public Collab so everybody can have a look.

Comment (njbruce) [reply src/assets/config_files/usecases.json]: @antonelepfl Sorry the "3D" at the start of the UUID was part of the HTML code in the file URL. I've updated usecases.json and it should be visible now.

**Comment by antonelepfl** [comment]
@njbruce I have deployed this on DEV environment.
@alex4200 Probably is a good idea if you can have a look to this UC and give your opinion.

**Comment by alex4200** [comment]
So far the notebook runs fine, just some minor spelling errors found in the text.Waiting for reply from @njbruce .

**Pull request merged**